CHANGELOG.md

```diff
@@ -1,3 +1,65 @@
+## 1.6.0 / 2017-04-14
+
+* [CHANGE] Replaced the **experimental** remote write implementations for
+  various backends by a generic write interface with example bridge
+  implementation for various backends.
+* [FEATURE] New flag `-storage.local.target-heap-size` to tell Prometheus about
+  the desired heap size. This deprecates the flags
+  `-storage.local.memory-chunks` and `-storage.local.max-chunks-to-persist`,
+  which are kept for backward compatibility.
+* [FEATURE] Add `check-metrics` to `promtool` to lint metric names.
+* [FEATURE] Auto-tweak times between checkpoints to limit time spent in
+  checkpointing to 50%.
+* [FEATURE] Add Joyent Triton discovery.
+* [FEATURE] `Scrape-Timeout-Seconds` header in HTTP scrape requests.
+* [FEATURE] Remote read interface, including example for InfluxDB. **Experimental.**
+* [FEATURE] Enable Consul SD to connect via TLS.
+* [FEATURE] Marathon SD supports multiple ports.
+* [FEATURE] Marathon SD supports bearer token for authentication.
+* [FEATURE] Match addresses with or without declared ports in Kubernetes SD.
```

Comment:
Hm, that's only a small change in an example config file. That wording might confuse people?